docs/docs-content/cluster-templates/create-cluster-template-policies/maintenance-policy.md

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_label: "Maintenance Policies"
+title: "Maintenance Policies"
+description: "Test" # UPDATE
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["cluster templates", "policies"]
+---
+
+:::preview
+
+:::
+
+A maintenance policy is a mandatory [policy](./create-cluster-template-policies.md) used with
+[cluster templates](../cluster-templates.md). Maintenance policies determine when and how upgrades are executed on the
```

Comment:
```suggestion
A maintenance policy is a required component for
[cluster templates](../cluster-templates.md). Maintenance policies determine when and how upgrades are executed on the
```

Reply:
Maybe? I think its better for the user to understand that it's a building block of the template.